config/locales/en/activemodel.yml

```diff
@@ -574,12 +574,12 @@ en:
             valuable_items_value:
               blank: Enter the estimated total value of valuable items
               greater_than_or_equal_to: Estimated value must be £500 or more
-              not_a_number: Enter the amount for the worth valuable items, like 1,000 or 20.30
-              too_many_decimals: Estimated valuable items value must not include more than 2 decimal numbers
+              not_a_number: &valuable_items_value_error Enter the amount for the worth valuable items, like 1,000 or 20.30
```

Comment:
Sorry Katharine, I think this might be a previous type. but should this be 
```
Enter the amount for the worth of the valuable items, like 1,000 or 20.30
                               ^^^^^^
```
The current value doesn't make a lot of sense